model/src/main/scala/uk/gov/ons/addressIndex/model/AddressIndexModel.scala

```diff
@@ -3,13 +3,13 @@ package uk.gov.ons.addressIndex.model
 import java.util.UUID
 
 case class AddressIndexUPRNRequest(
-  format: AddressScheme,
+  format: Option[AddressScheme],
   uprn: BigInt,
   id: UUID
```

Comment:
It'll be used in logging info about requests - so we can track a request all the way through the system